hw/ip/prim_xilinx/prim_xilinx_ram_1p.core

```diff
@@ -9,7 +9,6 @@ filesets:
   files_rtl:
     depend:
       - lowrisc:prim:assert
-      - lowrisc:prim:ram_1p_pkg
```

Comment:
Note for other reviewers: `lowrisc:prim_generic:ram_1p` is already included below.

Reply:
This dependency also wouldn't work properly anymore anyway. Our fusesoc fork cannot run generators recursively, so the only way the prim library-specific cores of `lowrisc:prim:ram_1p_pkg` get included is if some non-generated core includes it.